pkg/operator/operator.go

```diff
@@ -72,9 +72,9 @@ const (
 
 	// The official images to be used with this version of the operator. For debugging
 	// and emergency use cases they may be overwritten through options.
-	ImageCollector      = "gke.gcr.io/prometheus-engine/prometheus:v2.28.1-gmp.5-gke.0"
-	ImageConfigReloader = "gke.gcr.io/prometheus-engine/config-reloader:v0.3.1-gke.0"
-	ImageRuleEvaluator  = "gke.gcr.io/prometheus-engine/rule-evaluator:v0.3.1-gke.0"
+	ImageCollector      = "gke.gcr.io/prometheus-engine/prometheus:v2.28.1-gmp.7-gke.0"
+	ImageConfigReloader = "gke.gcr.io/prometheus-engine/config-reloader:v0.3.3-gke.0"
+	ImageRuleEvaluator  = "gke.gcr.io/prometheus-engine/rule-evaluator:v0.3.3-gke.0"
 
 	// The k8s Application, will be exposed as component name.
 	KubernetesAppName    = "app"
```

Comment:
You'll also want to bump the user agent string: https://github.com/GoogleCloudPlatform/prometheus-engine/blob/main/pkg/export/export.go#L451